pkg/services/ngalert/schedule/testing.go

```diff
@@ -239,32 +239,32 @@ func (f *fakeRuleStore) UpdateRuleGroup(cmd store.UpdateRuleGroupCmd) error {
 	return nil
 }
 
-type fakeInstanceStore struct {
+type FakeInstanceStore struct {
```

Comment:
I think it is time to move it out of schedule package to store along with other stores. Ideally, I would like to make mocks out of it.